{{cookiecutter.repostory_name}}/devops/tf/main/files/docker-compose.yml

```diff
@@ -45,7 +44,7 @@ services:
     logging: &exporter_logging
       driver: journald
       options:
-        tag: '{% raw %}{{.Name}}{% endraw %}'
+        tag: '{% raw %}{###{.Name}###}{% endraw %}'
```

Comment:
question: the previous value seemed to be correct; why was it changed?

Reply:
the previous value never worked, it broke on `terraform apply`